analysis/src/BuildSystem.ml

```diff
@@ -5,19 +5,42 @@ let namespacedName namespace name =
 
 let ( /+ ) = Filename.concat
 
+(*
+Here is the real problem:
```

Comment:
@zth I would use an environment variable here to avoid breaking all the analysis commands.
Thoughts?